system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/overview/SidePanelContent.js

```diff
@@ -0,0 +1,292 @@
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/* eslint react/display-name: "off" */
+
+import "./index.css";
+import Avatar from "@material-ui/core/Avatar";
+import Blue from "@material-ui/core/colors/blue";
+import BlurOnIcon from "@material-ui/icons/BlurOn";
+import CellIcon from "@material-ui/icons/CheckBoxOutlineBlank";
+import CheckCircleOutline from "@material-ui/icons/CheckCircleOutline";
+import ErrorIcon from "@material-ui/icons/ErrorOutline";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import ExpansionPanel from "@material-ui/core/ExpansionPanel";
+import ExpansionPanelDetails from "@material-ui/core/ExpansionPanelDetails";
+import ExpansionPanelSummary from "@material-ui/core/ExpansionPanelSummary";
+import Grain from "@material-ui/icons/Grain";
+import Grey from "@material-ui/core/colors/grey";
+import HttpTrafficIcon from "@material-ui/icons/CallSplit";
+import IconButton from "@material-ui/core/IconButton";
+import {Link} from "react-router-dom";
+import MUIDataTable from "mui-datatables";
+import Orange from "@material-ui/core/colors/orange";
+import PropTypes from "prop-types";
+import React from "react";
+import StateHolder from "../common/state/stateHolder";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Timeline from "@material-ui/icons/Timeline";
+import Typography from "@material-ui/core/Typography";
+import withGlobalState from "../common/state";
+import {withStyles} from "@material-ui/core/styles";
+import withColor, {ColorGenerator} from "../common/color";
+
+
+const styles = (theme) => ({
+    drawerContent: {
+        padding: 20
+    },
+    sideBarContentTitle: {
+        fontSize: 14,
+        fontWeight: 500,
+        display: "inline-flex",
+        paddingLeft: 5
+    },
+    titleIcon: {
+        verticalAlign: "middle"
+    },
+    sidebarTableCell: {
+        padding: 10
+    },
+    avatar: {
+        width: 25,
+        height: 25,
+        fontSize: 10,
+        fontWeight: 600,
+        color: "#fff"
+    },
+    sidebarContainer: {
+        marginBottom: 30
+    },
+    expansionSum: {
+        padding: 0,
+        borderBottomWidth: 1,
+        borderBottomStyle: "solid",
+        borderBottomColor: Grey[300]
+    },
+    cellIcon: {
+        verticalAlign: "middle"
+    },
+    panel: {
+        marginTop: 15,
+        boxShadow: "none",
+        borderTopWidth: 1,
+        borderTopStyle: "solid",
+        borderTopColor: Grey[200]
+    },
+    secondaryHeading: {
+        paddingRight: 10
+    },
+    panelDetails: {
+        padding: 0,
+        marginBottom: 100
+    },
+    sidebarListTableText: {
+        fontSize: 12
+
```

Comment:
unwanted blank line

Reply:
Removed in development repo https://github.com/wso2-cellery/mesh-observability/blob/master/components/global/portal/io.cellery.observability.ui/src/components/overview/SidePanelContent.js